### ISSUE TITLE
Multiple weird errors after install of @unocss/nuxt (Nuxt3 rc6)

### ISSUE THREAD

**Issue by AndreaFYB** [unocss/unocss#1585]
I'm very confused at what's causing this because the error doesn't feel like it makes sense. To explain, I have a normally functioning website - and I wanted to use the icons within `unocss` instead of continuing to use font-awesome. However, I kept running into a weird issue.

Whenever I install `@unocss/nuxt` (`"^0.45.21"`), something in vite blows up - I get the following errors:

```
 ERROR  11:42:51 [vite] Internal server error: ENOENT: no such file or directory, open '/_nuxt/components/Metric.vue'
 ...
 ERROR  11:42:51 [vite] Internal server error: ENOENT: no such file or directory, open '/_nuxt/components/Ticker.vue'
 ...
```

These are _local_ components that I wrote myself and are being imported directly from `components` - and these errors only appear after I _install_ `@unocss/nuxt` - not even when adding it as a plugin. That's what I find weird.

If I just install the dependency, without even adding it as a plugin or doing anything else, these errors appear. If I add the plugin, same thing happens. The moment I remove the dependency - it suddenly works again. If I try installing `unocss` on its own, things keep working - which makes me think that something is weird specifically with `@unocss/nuxt`.

In addition to these errors, I also get the following in the browser console:

```
GET http://localhost:3000/_nuxt/node_modules/@nuxtjs/tailwindcss/dist/runtime/tailwind.css
[HTTP/1.1 404 Not Found 10ms]

GET http://localhost:3000/_nuxt/node_modules/@fortawesome/fontawesome-svg-core/styles.css
[HTTP/1.1 404 Not Found 9ms]

GET http://localhost:3000/_nuxt/assets/css/root.css
[HTTP/1.1 404 Not Found 10ms]

GET http://localhost:3000/_nuxt/@vite/client
[HTTP/1.1 404 Not Found 9ms]

GET http://localhost:3000/_nuxt/entry.mjs
[HTTP/1.1 404 Not Found 9ms]
```

In other words, it seems that the moment I install this dependency, a bunch of `css` becomes inaccessible - with `styles.css` and `root.css` specifically being under the `css` property in `defineNuxtConfig`. Not sure why this is happening - but it might be a good direction to what's going wrong. I'd think that maybe UnoCSS is incompatible with Tailwind/FortAwesome but my custom `root.css` is breaking too...

Any ideas? If you need more information let me know.
